runtime/src/bank.rs

```diff
@@ -5222,33 +5237,49 @@ impl Bank {
     /// update bank's rewrites set for all rewrites that were skipped
     /// if 'just_rewrites', function will only update bank's rewrites set and not actually store any accounts
     /// return # accounts loaded
-    fn collect_rent_in_partition(&self, partition: Partition, just_rewrites: bool) -> usize {
+    fn collect_rent_in_partition(
+        &self,
+        partition: Partition,
+        just_rewrites: bool,
+        metrics: &RentMetrics,
+    ) -> usize {
         let subrange = Self::pubkey_range_from_partition(partition);
 
+        let mut hold_range = Measure::start("hold_range");
         let thread_pool = &self.rc.accounts.accounts_db.thread_pool;
         self.rc
             .accounts
             .hold_range_in_memory(&subrange, true, thread_pool);
+        hold_range.stop();
```

Comment:
we aren't using the result of this measure, can we remove it or should it be added to the `RentMetrics`?

Reply:
added